### ISSUE TITLE
python2 Installation Not Installing symlink To pip2

### ISSUE THREAD

**Issue by tundratim** [Linuxbrew/homebrew-core#6516]
Until recently, when I did "brew install python", I got python2 and pip2 installed.  Now it installs python3 and pip3.
No big deal, I changed my makefile to now explicitly install python and python2 (it used to install python and python3).   This works, but the symlink to pip3 gets installed in ./bin, but not the symlink to pip2.

I believe this is just an installation bug.

**Comment by iMichka** [comment]
Yes, and we are changing back to the old behaviour, because people got too much upset about python now being python3, and this is not PEP 394 compliant. I will merge the needed commits in a few hours, so you may try again a little bit later. When everything is stabilised we can then iron out the last bugs. Sorry for the disagreement.

**Comment by iMichka** [comment]
Can you try again?

**Comment by tundratim** [comment]
@iMichka Confirmed.  The build now properly installs the symlinks.

There is, however, one small change in behavior I noticed.  This is not a problem, I just wanted to confirm that this was your intent.  Previously, a python installation would install symlinks for python2/3 and pip2/3.  It was up to me to link them to python on pip.  I noticed that in this install, the python/pip symlinks were setup by the installer, presumably for the first python instance it processed.  Was this intentional?

**Comment by tseemann** [comment]
I tend to do this when things go weird:
```
brew reinstall python3
brew reinstall python
alias pip='echo '\''please use pip2 or pip3 instead'\'''
```
